packages/common/src/validations/password_schema.ts

```diff
@@ -4,8 +4,10 @@ import * as yup from "yup";
 // has a minimum length of 8 characters
 export const passwordSchema = yup
     .string()
+    .min(6)
```

Comment:
Do you not need to add there error message here as well?
```suggestion
    .min(6, "Password must be between 6 and 20 characters.")
```

Reply:
Yup, you're right. It's fixed